src/themes/admin_future/modules/comment/config.tpl

```diff
@@ -0,0 +1,81 @@
+                        <th class="text-nowrap">{$LANG->getModule('auto_postcomm')}</th>
+                        <th class="text-nowrap">{$LANG->getModule('emailcomm')}</th>
+                        <th class="text-nowrap">{$LANG->getModule('funcs')}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {assign var="WEIGHT" value=1}
+                    {foreach $SITE_MOD_COMM as $MOD => $ROW}
+                    {if !empty($MODULE_CONFIG.$MOD.allowed_comm)}
+                        {assign var="ARRAY_ALLOWED_COMM" value="intval"|array_map:($MODULE_CONFIG.$MOD.allowed_comm|split:',')}
+                    {else}
+                        {assign var="ARRAY_ALLOWED_COMM" value=[]}
+                    {/if}
+                    {if (-1)|in_array:$ARRAY_ALLOWED_COMM:true}
+                        {append var="ROW" value=$LANG->getModule('allowed_comm_item') index="allowed_comm"}
+                    {else}
+                        {assign var="ALLOWED_COMM" value=[]}
+                        {foreach $ARRAY_ALLOWED_COMM as $GID}
+                            {append var="ALLOWED_COMM" value=$GROUPS.$GID}
+                        {/foreach}
+                        {append var="ROW" value=$ALLOWED_COMM|join:"<br>" index="allowed_comm"}
+                    {/if}
+
+                    {if !empty($MODULE_CONFIG.$MOD.view_comm)}
+                        {assign var="ARRAY_VIEW_COMM" value=$MODULE_CONFIG.$MOD.view_comm|split:','}
+                    {else}
+                        {assign var="ARRAY_VIEW_COMM" value=[]}
+                    {/if}
+                    {assign var="VIEW_COMM" value=[]}
+                    {foreach $ARRAY_VIEW_COMM as $GID}
+                        {append var="VIEW_COMM" value=$GROUPS.$GID}
+                    {/foreach}
+                    {append var="ROW" value=$VIEW_COMM|join:"<br>" index="view_comm"}
+                    {if !empty($ROW.admin_title)}
+                        {append var="ROW" value=$ROW.admin_title index="custom_title"}
+                    {/if}
+                    <tr>
+                        <td class="text-center">{$WEIGHT}</td>
+                        <td>{$ROW.custom_title}</td>
+                        <td class="text-center"><i class="fa-solid fa-{if $MODULE_CONFIG.$MOD.activecomm}check{else}xmark{/if} fa-lg"></i></td>
+                        <td>{$ROW.allowed_comm}</td>
+                        <td>{$ROW.view_comm}</td>
+                        <td>{$LANG->getModule('auto_postcomm_'|cat:$MODULE_CONFIG.$MOD.auto_postcomm)}</td>
+                        <td class="text-center"><i class="fa-solid fa-{if $MODULE_CONFIG.$MOD.emailcomm}check{else}xmark{/if} fa-lg"></i></td>
+                        <td class="text-center text-nowrap"><button class="btn btn-secondary" data-mod="{$MOD}"><i class="fa-solid fa-edit fa-lg"></i>&nbsp;{$LANG->getModule('edit')}</button></td>
```

Comment:
@danlt150 anh quên chỗ này:

- Icon đổi giùm anh thành `<i class="fa-solid fa-pencil"></i>` thống nhất cho mọi nút sửa. Tại trước đây font-awesome 4.x nó không có icon đó, chỉ có edit, sau bổ sung thêm nhìn nó nhẹ nhàng hài hòa hơn
- Bỏ fa-lg vì nhìn icon to lắm
- Bên main.tpl tiện đổi giùm anh luôn